### PR TITLE
Feat/add localization

### PR DESCRIPTION
**Pull request by p-pavlo**

### CODE REVIEW

File: src/constants/locales.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
...
-export const LOCALE_NAME_UA = 'UA';
-export const LOCALE_NAME_RU = 'RU';
-export const LOCALE_NAME_PL = 'PL';
+export const LOCALE_NAME_EN = 'en';
+export const LOCALE_NAME_UA = 'ua';
+export const LOCALE_NAME_RU = 'ru';
+export const LOCALE_NAME_PL = 'pl';
+export const ORIGINAL_LANGUAGE_NAME_EN = 'English';
+export const ORIGINAL_LANGUAGE_NAME_UA = 'Українська';
+export const ORIGINAL_LANGUAGE_NAME_RU = 'Русский';
```
Comment (Densmirnov12) [inline src/constants/locales.js]: exports in cascade order

File: src/icons/index.js [hunk 2]
```diff
@@ -187,5 +187,80 @@ export const Logout = ({ color, width, height }) => (
...
+      <path fill='#22408C' d='M29.942 14.514H0V6.486h29.942z' />
+      <path
+        fill='#EEE'
+        d='M26.615.063H3.327C1.49.063 0 1.5 0 3.274v3.212h29.942V3.274c0-1.773-1.49-3.211-3.327-3.211'
+      />
+    </g>
+  </svg>
+);
+
+export const Tick = () => (
```
Comment (VitaliyAndrushko) [inline src/icons/index.js]: ({ color, width, height }) and add prop types

File: src/pages/language/index.js [hunk 3]
```diff
@@ -1,15 +1,106 @@
...
+        ))}
+      </Flex>
+    </PageWrapper>
+  );
 };
 
 export default LanguagePage;
 
-LanguagePage.propTypes = {};
+LanguagePage.propTypes = {}; // TODO: check what prop types you have on react devtools and set here
```
Comment (VitaliyAndrushko) [inline src/pages/language/index.js]: add prop types

File: src/ui/wrappers/index.js [hunk 4]
```diff
@@ -25,3 +25,9 @@ export const AuthPagesWrapper = styled.div`
 `;
 
 AuthPagesWrapper.displayName = 'AuthPagesWrapper';
+
+// TODO: use color from Theme
+export const PageWrapper = styled.div`
+  height: 100vh;
+  background-color: #1f2532;
```
Comment (VitaliyAndrushko) [inline src/ui/wrappers/index.js]: color from Theme and remove TODO comment

File: src/icons/index.js [hunk 5]
```diff
@@ -187,5 +187,80 @@ export const Logout = ({ color, width, height }) => (
     />
   </svg>
 );
+
+export const UkrFlag = () => (
+  <svg xmlns='http://www.w3.org/2000/svg' width='30' height='22' viewBox='0 0 30 22'>
+    <g fill='none' fill-rule='evenodd'>
```
Comment (Densmirnov12) [inline src/icons/index.js]: rename prop fill-rule => fillRule

File: src/icons/index.js [hunk 6]
```diff
@@ -187,5 +187,80 @@ export const Logout = ({ color, width, height }) => (
     />
   </svg>
 );
+
+export const UkrFlag = () => (
```
Comment (Densmirnov12) [inline src/icons/index.js]: export const UkrFlag = ({ color, width, height }) => () and set this props

File: src/icons/index.js [hunk 7]
```diff
@@ -187,5 +187,80 @@ export const Logout = ({ color, width, height }) => (
...
+        fill='#005BBB'
+        d='M26.64.065H3.33A3.327 3.327 0 0 0 0 3.388v7.475h29.97V3.388A3.327 3.327 0 0 0 26.64.065'
+      />
+      <path
+        fill='#FFD500'
+        d='M29.97 18.34a3.327 3.327 0 0 1-3.33 3.322H3.33A3.327 3.327 0 0 1 0 18.34v-7.476h29.97v7.476z'
+      />
+    </g>
+  </svg>
+);
```
Comment (Densmirnov12) [inline src/icons/index.js:216]: add proptypes and defaultprops

File: src/constants/locales.js (hunk 1 above)
Comment (p-pavlo) [reply src/constants/locales.js]: +

Comment (p-pavlo) [reply src/constants/locales.js]: +

File: src/icons/index.js [hunk 8]
```diff
@@ -189,3 +199,85 @@ export const Logout = ({ color, width, height }) => (
...
+RusFlag.propTypes = flagPropTypes;
+RusFlag.defaultProps = defaultFlagProps;
+
+export const Tick = ({ color, width, height }) => (
+  <svg xmlns='http://www.w3.org/2000/svg' width={width} height={height} viewBox='0 0 14 11'>
+    <path
+      fill={color}
+      fill-rule='evenodd'
```
Comment (Densmirnov12) [inline src/icons/index.js]: fillRule

File: src/locale/index.js [hunk 9]
```diff
@@ -2,10 +2,16 @@
 import * as C from '../constants';
 // /////////////////////////////////////////////////////////////////////////////////////////////////
 
-const UA = {
+export const UA = {
   termsAndConditions: 'Умови використання',
   privacyPolicy: 'Політика приватності',
   copyright: '2019 © Всі права захищено',
+  localeNames: {
```
Comment (Densmirnov12) [inline src/locale/index.js:9]: use cascade order, if this object did not map

**Review by Densmirnov12 — APPROVED** [review]

File: src/helpers/common.js [hunk 10]
```diff
@@ -11,15 +11,20 @@ import * as H from './';
 import locales from '../locale';
 // /////////////////////////////////////////////////////////////////////////////////////////////////
 
-export const getLocaleName = () =>
-  H.ifElse(
+export const getLocaleName = () => {
+  const localeString = H.ifElse(
     H.isNotNilAndNotEmpty(H.getItemFromLocalStorage('localeName')),
```
Comment (sancraz) [inline src/helpers/common.js:16]: create const localeNameFromLocalStorage = H.getItemFromLocalStorage('localeName') and use

File: src/locale/index.js [hunk 11]
```diff
@@ -2,10 +2,16 @@
...
+    [C.LOCALE_NAME_UA]: 'Українська',
+    [C.LOCALE_NAME_EN]: 'Англійська',
```
Comment (sancraz) [inline src/locale/index.js:11]: Maybe better not to translate languages. After accident change of language user might be confused how to select previous language if he does not know how it looks on current

**Pull request merged**